modules/github_integration/lib/open_project/github_integration/engine.rb

```diff
@@ -37,7 +37,7 @@ class Engine < ::Rails::Engine
     include OpenProject::Plugins::ActsAsOpEngine
 
     register 'openproject-github_integration',
-             :author_url => 'http://finn.de',
+             :author_url => 'https://openproject.com',
```

Comment:
shouldn't it be www.openproject.org?

Reply:
true (it's a redirect and works, but www.openproject.org seems to be the final destination :)

```suggestion
             :author_url => 'https://www.openproject.org/',
```